modules/react/checkbox/lib/CheckBackground.tsx

```diff
@@ -24,17 +24,17 @@ const checkboxBackgroundStencil = createStencil({
     position: 'absolute',
     transition: 'border 200ms ease, background 200ms',
     width: calc.add(system.space.x4, px2rem(2)),
-    border: `${px2rem(1)} solid ${base.licorice200}`,
+    border: `${px2rem(1)} solid ${system.color.fg.muted.soft}`,
```

Comment:
Is this supposed to be using the `border` system token? If this is missing, should we use `base` for now until it's updated.